targets/targets.json

```diff
@@ -8845,6 +8845,74 @@
         "release_versions": ["5"],
         "device_name": "STM32L151CBxxA"
     },
+    "NUMAKER_IOT_M263A": {
+        "core": "Cortex-M23",
+        "default_toolchain": "ARMC6",
+        "extra_labels": [
+            "NUVOTON",
+            "M261",
+            "M263KIAAE",
+            "FLASH_CMSIS_ALGO"
+        ],
+        "macros": [
+            "MBED_FAULT_HANDLER_DISABLED",
```

Comment:
It's disabled by default. But what's the reason behind this flag here?

Reply:
@bulislaw , it's `mbed_fault_handler.c`  not support Cortex-M23 in early June. However, last `mbed_fault_handler.c` seems consider TARGET_M23. Let me do some verification.

Reply:
@bulislaw , After verification, it could support Cortex-M23 and not disabled it by default. Please see the commit 70c8a3b